hr_worked_days_from_timesheet/models/hr_payslip.py

```diff
@@ -18,23 +19,17 @@ def _timesheet_mapping(self, timesheet_sheets, payslip, date_from,
         module and creates a dict of worked days to be created in the payslip.
         """
         # Create one worked days record for each timesheet sheet
-        wd_model = self.env['hr.payslip.worked_days']
-        uom_obj = self.env['product.uom']
-        uom_hours = self.env.ref('product.product_uom_hour')
         for ts_sheet in timesheet_sheets:
             # Get formated date from the timesheet sheet
             date_from_formated = fields.Date.to_string(
                 fields.Datetime.from_string(ts_sheet.date_from))
             number_of_hours = 0
             for ts in ts_sheet.timesheet_ids:
                 if date_from <= ts.date <= date_to:
-                    unit_amount = uom_obj._compute_qty_obj(
-                        ts.product_uom_id,
-                        ts.unit_amount, uom_hours)
+                    unit_amount = ts.unit_amount
                     number_of_hours += unit_amount
-
```

Comment:
Couldn't be written, more synthetically, as: `number_of_hours += ts.unit_amount` and erasing the previous line?